isthmus/src/main/java/io/substrait/isthmus/expression/RexExpressionConverter.java

```diff
@@ -153,4 +280,20 @@ public Expression visitLocalRef(RexLocalRef localRef) {
   public Expression visitPatternFieldRef(RexPatternFieldRef fieldRef) {
     throw new UnsupportedOperationException("RexPatternFieldRef not supported");
   }
+
+  public RelNode getInputRel() {
```

Comment:
What's the purpose of code block L24 - 297, for a `RexExpressionConverter`?

Reply:
benefit line 119, for the invocator to set or reset the window expression's related relnode and type